nav2_map_server/include/nav2_map_server/occ_grid_loader.hpp

```diff
@@ -121,6 +150,9 @@ class OccGridLoader : public rclcpp_lifecycle::node_interfaces::LifecycleNodeInt
   // The message to publish on the occupancy grid topic
   std::unique_ptr<nav_msgs::msg::OccupancyGrid> msg_;
 
+  // Parameters of last loaded map
+  LoadParameters load_parameters_;
+
   // The frame ID used in the returned OccupancyGrid message
   static constexpr const char * frame_id_{"map"};
```

Comment:
I know you didn't modify this, but can you make this a string as well? This should be a parameter option on startup so it should be trivially changable

Reply:
Fixed: https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-95ed568d688f8e5356bd745ad04513a3R128, https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-e5c96c9122118a911c895ffa5dbd7b00R47